pkg/lib/serverTime.js

```diff
@@ -258,42 +265,27 @@
         return status;
     };
 
-    const custom_ntp_config_file = cockpit.file("/etc/systemd/timesyncd.conf.d/50-cockpit.conf",
-                                                { superuser: "try" });
+    function get_timesync_backend() {
+        // If chronyd is installed, we assume that it takes precedence
+        // over timesyncd.
+        if (chronyd_service.exists)
+            return "chronyd";
+        else if (timesyncd_service.exists)
+            return "timesyncd";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17992-20230110-142000-984810ea-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)